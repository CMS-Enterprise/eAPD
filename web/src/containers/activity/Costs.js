import PropTypes from 'prop-types';
import React from 'react';

import Expenses from './NonPersonnelCosts';
import StatePersonnel from './StatePersonnel';
import { Subsection } from '../../components/Section';
import { t } from '../../i18n';

const Costs = ({ aKey }) => (
  <Subsection resource="activities.costs" nested>
    <h5 className="ds-h4">{t('activities.costs.subtitles.personnel')}</h5>
    <StatePersonnel aKey={aKey} />
    <hr />

    <h5 className="ds-h4">{t('activities.costs.subtitles.nonPersonnel')}</h5>
    <Expenses aKey={aKey} />
    <hr />
  </Subsection>
);

Costs.propTypes = {
  aKey: PropTypes.string.isRequired
};

export default Costs;
