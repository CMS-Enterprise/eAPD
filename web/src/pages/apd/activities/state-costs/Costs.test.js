import { shallow } from 'enzyme';
import React from 'react';

import Costs from './Costs';

describe('activity costs wrapper (around personnel and non-personnel) ', () => {
  it('renders properly', () => {
    expect(
      shallow(<Costs activityIndex={0} aKey="activity key" />)
    ).toMatchSnapshot();
  });
});
