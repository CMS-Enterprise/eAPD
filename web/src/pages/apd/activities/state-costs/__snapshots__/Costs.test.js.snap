// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`activity costs wrapper (around personnel and non-personnel)  renders properly 1`] = `
<Subsection
  headerClassName={null}
  id="activity-cost-categories-0"
  nested={false}
  resource="activities.costs"
>
  <h4
    className="ds-h4"
  >
    State Staff
  </h4>
  <Connect(StatePersonnel)
    activityIndex={0}
  />
  <h4
    className="ds-h4"
  >
    Other State Expenses
  </h4>
  <Connect(NonPersonnelCosts)
    activityIndex={0}
  />
</Subsection>
`;
